catroid/src/main/java/org/catrobat/catroid/ui/ProgramMenuActivity.java

```diff
@@ -82,7 +82,7 @@ public void onCreate(Bundle savedInstanceState) {
 			Sprite sprite = ProjectManager.getInstance().getCurrentSprite();
 			Scene scene = ProjectManager.getInstance().getCurrentScene();
 			if (sprite != null && scene != null && actionBar != null) {
-				String title = scene.getName() + ": " + sprite.getName();
+				String title = ProjectManager.getInstance().getCurrentProject().getSceneList().size() > 1 ? scene.getName() + ": " + sprite.getName() : sprite.getName();
```

Comment:
@AyyalAwwad is concerning with RTL support but this will probably still last some time